Userland/Applications/Escalator/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+serenity_component(
+    Escalator
+    REQUIRED
+    TARGETS Escalator
+)
+
+compile_gml(Escalator.gml EscalatorGML.h escalator_gml)
+
+set(SOURCES
+    main.cpp
+    EscalatorWindow.cpp
+    EscalatorWindow.h
+    EscalatorGML.h
```

Comment:
We usually don't put header files in `SOURCES` unless there is a reason for that.

Reply:
Looks like EscalatorGML.h will still be required here, but not EscalatorWindow.h. This is so that symbol `escalator_gml` can be used when setting up the window. I will go ahead and remove EscalatorWindow.h.